.github/ISSUE_TEMPLATE/feature-request.yml

```diff
@@ -0,0 +1,38 @@
+name: "\U0001F680 Feature / enhancement request"
+description: Suggest a feature or an enhancement you'd like to see in Iroha
+title: "[suggestion] "
+labels: [ "Enhancement", "iroha2" ]
+body:
+  - type: textarea
+    id: feature-request
+    attributes:
+      label: Feature request
+      description: |
+        Please describe a feature you'd like to see in Iroha in detail. Provide examples and links to explain the idea, if possible.
+      placeholder: |
+        I'd like Iroha 2 to …
+    validations:
+      required: true
+  - type: textarea
+    id: motivation
+    attributes:
+      label: Motivation
+      description: |
+        Outline the motivation for adding a new feature. If this is related to other issues, adding a link to those issues will be helpful.
+      placeholder: |
+        Iroha will be even more useful if …
+    validations:
+      required: true
+  - type: input
+    id: who-can-help
+    attributes:
+      label: Who can help?
+      description: |
+        If you figure out the right person to tag, your issue might be solved faster.
+        You can use `git blame` or check the following list of people **you can tag** based on what the issue is connected to:
```

Comment:
does this make sense in the context of asking for a new feature?

Reply:
Let's use:

> You can **tag** people in the following list based on the issue topic

It is still important to tag someone, so that they get a notification on a feature proposal.

Reply:
ok
**from** the following list